python/cudf/cudf/utils/dtypes.py

```diff
@@ -640,6 +643,35 @@ def _can_cast(from_dtype, to_dtype):
         return np.can_cast(from_dtype, to_dtype)
 
 
+def _map_to_default_dtypes(inferred_dtypes: List[np.dtype]) -> List[np.dtype]:
```

Comment:
Wouldn't it be more flexible for this function to accept a single `dtype` and return a single `dtype`?

Reply:
I thought about that too, but also I don't want to build the `dtype_map` on every call since that's too much overhead. Any thoughts?

Reply:
No longer applicable due to recent update.